x-pack/platform/plugins/private/observability_ai_assistant_management/public/routes/components/settings_tab/product_doc_entry.tsx

```diff
@@ -22,16 +22,18 @@ import { useKibana } from '../../../hooks/use_kibana';
 import { useGetProductDocStatus } from '../../../hooks/use_get_product_doc_status';
 import { useInstallProductDoc } from '../../../hooks/use_install_product_doc';
 import { useUninstallProductDoc } from '../../../hooks/use_uninstall_product_doc';
+import { useInferenceId } from './inference_id_context';
 
 export function ProductDocEntry() {
   const { overlays } = useKibana().services;
+  const { selectedInferenceId } = useInferenceId();
```

Comment:
Can you use
```suggestion
  const knowledgeBase = useKnowledgeBase();
```

and then `knowledgeBase.status.value.currentInferenceId`?

Reply:
I suggest going with `const knowledgeBase = useKnowledgeBase();`. It's the simpler option and I don't believe the UX is worse.